papers/ReTraCk/parser/sparql_executor.py

```diff
@@ -6,7 +6,7 @@
 
 from SPARQLWrapper import SPARQLWrapper, JSON, SPARQLExceptions
 
-sparql = SPARQLWrapper("http://localhost:8890/sparql/")
+sparql = SPARQLWrapper("http://localhost:3001/sparql/")
```

Comment:
Isn't this port set in some config? We should probably read from there.

Reply:
No, not in config. 

[bash $RETRACK_HOME/scripts/virtuoso.sh](https://github.com/microsoft/KC/blob/48ba13784d63054eda6f2b68728e4369eca2804e/papers/ReTraCk/README.md?plain=1#L110) reports error in my side. 

So I use [python virtuoso.py start 3001 -d ./virtuoso_db](https://github.com/microsoft/KC/blob/48ba13784d63054eda6f2b68728e4369eca2804e/papers/ReTraCk/README.md?plain=1#L117), which start virtuoso at 3001.

Reply:
What's the error you get for that script? I suggest we add a second script with your way to start to be used in case the 1st fails.

Reply:
The error is /ReTraCk/scripts/virtuoso.sh: line 6: cd: /Freebase-Setup/virtuoso-opensource/db: No such file or directory

The current readme provides a second way to start virtuoso, which is just [python virtuoso.py start 3001 -d ./virtuoso_db](https://github.com/microsoft/KC/blob/48ba13784d63054eda6f2b68728e4369eca2804e/papers/ReTraCk/README.md?plain=1#L117).